cli/js/ops/net.ts

```diff
@@ -22,8 +22,8 @@ export enum ShutdownMode {
   ReadWrite, // unused
 }
 
-export function shutdown(rid: number, how: ShutdownMode): void {
-  sendSync("op_shutdown", { rid, how });
+export async function shutdown(rid: number, how: ShutdownMode): Promise<void> {
+  return sendSync("op_shutdown", { rid, how });
```

Comment:
@ry You said to make this `async`. Actually make the op async or just the exposed function for futureproofing?

Reply:
Future proofing is fine. Ultimately the op should be async.